<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WReport</class>
 <widget class="QWidget" name="WReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="Stack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>670</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Generator">
    <widget class="QRadioButton" name="In">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>46</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrada</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clean">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>574</y>
       <width>215</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Clean
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
    </widget>
    <widget class="QLabel" name="TitleGenerator">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt;</string>
     </property>
     <property name="text">
      <string>Gerar Relatorio</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ValueBox_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ValueBox_2
{
	color: black;
	border: none;
}</string>
     </property>
     <property name="maximum">
      <double>99999999999999995287335453651211007997446182781858083179085387749785952239205787068995699003416510776387310061494932420984963311567802202010637287727642443776.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="AccChoose">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>128</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolha as contas:</string>
     </property>
    </widget>
    <widget class="QLabel" name="RangeOfDate_2">
     <property name="geometry">
      <rect>
       <x>317</x>
       <y>72</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="TypesChoose">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>273</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolha os Tipos de Lancamentos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="RangeOfValues">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Faixa de valores:        De:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ValueBox_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ValueBox_1
{
	color: black;
	border: none;
}</string>
     </property>
     <property name="maximum">
      <double>99999999999999995287335453651211007997446182781858083179085387749785952239205787068995699003416510776387310061494932420984963311567802202010637287727642443776.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="Erro">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Erro
{
    color: red;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="Out">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>46</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Saida</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateEdit_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#DateEdit_1
{
	border: none;
}</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="AccTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>441</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Confirm">
     <property name="geometry">
      <rect>
       <x>257</x>
       <y>574</y>
       <width>215</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Confirm
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
    <widget class="QLabel" name="PayChoose">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>416</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolha os Tipos de Pagamentos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="RangeOfDate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>72</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Periodo:                        De:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="TypesTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>293</y>
       <width>441</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="PayTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>435</y>
       <width>441</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="RangeOfValues_2">
     <property name="geometry">
      <rect>
       <x>317</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Operation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>46</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Operacao:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DateEdit_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#DateEdit_2
{
	border: none;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllAccs">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#AllAccs
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Todas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllTypes">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>293</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#AllTypes
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Todos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllPay">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>435</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#AllPay
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Todos</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Report">
    <widget class="QLabel" name="TitleReport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt;</string>
     </property>
     <property name="text">
      <string>Relatorio</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalRel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancamentos Realizados:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Rep_Releases">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>113</y>
       <width>480</width>
       <height>250</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalMont">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total do montante das contas avaliadas:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rep_Back">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>441</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Rep_Back
{
    background-color: #a3f48a;
    color: black;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QWidget" name="WChart" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>371</y>
       <width>481</width>
       <height>242</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total de entradas:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalOut">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total de saidas:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalMont_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalIn_2">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalOut_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Rep_TotalRel_2">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
